lib/elixir/pages/cheatsheets/enum-cheat.cheatmd

```diff
@@ -431,8 +431,8 @@ iex> for item <- cart,
 ### [`into(enum, collectable)`](`Enum.into/2`)
 
 ```elixir
-iex> pairs = [{"apple", 3}, {"banana", 1}, {"orange", 6}]
-iex> Enum.into(pairs, %{})
+iex> cart = [{"apple", 3}, {"banana", 1}, {"orange", 6}]
```

Comment:
I think this is confusing, cart is defined at the top of the file and is shared in most functions. This example requires a different type, the name reflects this.